### PR TITLE
Adds vore

### PR DESCRIPTION
**Pull request by Accinator50**
## What Does This PR Do
Adds the vore ability to ~~Vulpkanins, Unathis and Tajarans~~ EVERY SPECIES!.

V - Vox
O - Eating
R - Ability
E - ! 

## Why It's Good For The Game
These tiny little bastards are barely the size of a mouse. Don't discriminate against other species by not letting them eat! It also drastically reduces production costs for KFV because they no longer have to fry their food.

## Images of changes

https://user-images.githubusercontent.com/44909173/229183962-140bb9a5-9f77-4c7e-8142-acb933176a79.mp4

## Testing
Ran it.

### CODE REVIEW

**Comment by QuacksQ** [comment]
Bro what

**Comment by Accinator50** [comment]
> 

Found the CEO of Donk Pocket. Guess you can't reduce your production costs that easily, hm? 
grafik

**Comment by SynthTwo** [comment]
me when i fucking get you

File: code/modules/mob/mob_grab.dm [hunk 1]
```diff
@@ -394,14 +394,32 @@
...
+    V:::::V     V:::::V    O:::::O     O:::::O  R:::::::::::::RR    E:::::::::::::::E
+     V:::::V   V:::::V     O:::::O     O:::::O  R::::RRRRRR:::::R   E:::::::::::::::E
+      V:::::V V:::::V      O:::::O     O:::::O  R::::R     R:::::R  E::::::EEEEEEEEEE
+       V:::::V:::::V       O:::::O     O:::::O  R::::R     R:::::R  E:::::E
+        V:::::::::V        O::::::O   O::::::O  R::::R     R:::::R  E:::::E       EEEEEE
+         V:::::::V         O:::::::OOO:::::::ORR:::::R     R:::::REE::::::EEEEEEEE:::::E
+          V:::::V           OO:::::::::::::OO R::::::R     R:::::RE::::::::::::::::::::E
+           V:::V              OO:::::::::OO   R::::::R     R:::::RE::::::::::::::::::::E
+            VVV                 OOOOOOOOO     RRRRRRRR     RRRRRRREEEEEEEEEEEEEEEEEEEEEE
+Shamelessy stolen trademark for Kentucky Fried Vox Inc. */
```
Comment (JimKil3) [inline code/modules/mob/mob_grab.dm:416]:
```suggestion
```

**Review by JimKil3 — CHANGES_REQUESTED** [review]

**Comment by AffectedArc07** [comment]
merging because vox nerfs are always allowed

**Pull request merged**